### ISSUE TITLE
Documentation - bulkscan_null()

### ISSUE THREAD

**Issue by harperkole** [senresearch/BulkLMM.jl#94]
should the documentation for the function mention that the h2 values are also returned, as well as the LOD scores?

**Comment by learningMalanya** [comment]
Yes. The documentation of subsequent functions have not been updated/done yet. The documentation for the general wrappers ("scan()", "bulkscan()") of the sub-functions gives the description of the output values and the description should be complete. Will finish the documentation of all other functions soon...

**Comment by learningMalanya** [comment]
> should the documentation for the function mention that the h2 values are also returned, as well as the LOD scores?

Yes. The documentation of subsequent functions have not been updated/done yet. The documentation for the general wrappers ("scan()", "bulkscan()") of the sub-functions gives the description of the output values and the description should be complete. Will finish the documentation of all other functions soon...
